datastore/importers/nmdb-import-arista-show-lldp-neighbor/Parser.cpp

```diff
@@ -0,0 +1,340 @@
+    qi::lit("Port")
+    > qi::lit("Neighbor Device ID")  
+    > qi::lit("Neighbor Port ID")
+    > qi::lit("TTL")
+    > qi::eol
+    > +qi::char_('-')
+    > qi::eol
+    ;
+
+  noDetailEntry =
+    token
+    > token [(pnx::ref(nd.curHostname) = qi::_1)]
+    > port  [(pnx::ref(nd.curIfaceName) = qi::_1)]
+    > qi::uint_
+    > qi::eol
+    ;
+
+  // ----- detail -----
+
+  detailConfig =
+    (detailHeader
+    >> (detailEntry | *qi::eol)
+    ) [(pnx::bind(&Parser::finalizeData, this))]
+    ;
+
+  detailHeader =
+    qi::lit("Interface")
+    >> port [(pnx::ref(nd.srcIfaceName) = qi::_1)]
+    >> qi::lit("detected")
+    > ignoredLine
+    > qi::eol
+    ;
+
+  detailEntry =
+    detailNeighborLine
+    > detailDiscovered
+    > +(detailChassisId
+      | detailPortId
+      | detailPortDescription
+      | detailSystemName
+      | detailSystemDescription
+      | detailCapabilities
+      | detailVlan
+      | (!( detailHeader
+          | detailNeighborLine
+          ) >> ignoredLine - qi::eol
+        )
+      )
+    ;
+
+  detailNeighborLine =
+    qi::lit("Neighbor")
+    > +(qi::char_ - '/')
+    > '/'
+    > ( inQuotes | port)
+    > qi::lit(", age")
+    > ignoredLine
+    ;
+
+  detailDiscovered =
+    qi::lit("Discovered")
+    > qi::uint_
+    > qi::lit("days,")
+    > token
+    > ignoredLine
+    ;
+
+  detailChassisId =
+    qi::lit("- Chassis ID type:") > restOfLine
+    > qi::lit("Chassis ID     :") > restOfLine [(pnx::ref(nd.curMacAddr) = qi::_1)]
+    ;
+
+  detailPortId =
+    qi::lit("- Port ID type:") > restOfLine
+    > qi::lit("Port ID     :") > inQuotes [(pnx::ref(nd.curIfaceName) = qi::_1)]
+    > qi::eol
+    ;
+
+  detailPortDescription =
+    qi::lit("- Port Description:")
+    > inQuotes [(pnx::ref(nd.curPortDescription) = qi::_1)]
+    > qi::eol
+    ;
+
+  detailSystemName =
+    qi::lit("- System Name:")
+    >> inQuotes [(pnx::ref(nd.curHostname) = qi::_1)]
+    > qi::eol
+    ;
+
+  detailSystemDescription =
+    qi::lit("- System Description:")
+    > inQuotes [(pnx::ref(nd.curSysDescription) = qi::_1)]
+    > qi::eol
+    ;
+
+  detailCapabilities =
+    (qi::lit("- System Capabilities :")
+    > restOfLine [(pnx::ref(nd.curDeviceType) = qi::_1)]
+    > -(qi::lit("Enabled Capabilities:"))
+    >> restOfLine) [(pnx::ref(nd.curDeviceType) = qi::_1)]
+    ;
+
+  detailVlan =
+    qi::lit("VLAN ID: ")
+    > qi::ushort_ [(pnx::bind(&Parser::addVlan, this, qi::_1))]
+    > qi::lit(", VLAN Name: ")
+    > inQuotes
+    > qi::eol
+    ;
+
+
+  // ----- common usage -----
+  port =
+    +(!qi::lit(',') >> qi::ascii::graph)
+    > -qi::hold[
+        (qi::char_(' ')
+        >> (+qi::ascii::digit >> *(qi::char_('/') >> +qi::ascii::digit))
+        )]
+    ;
+
+  restOfLine =
+    *(qi::char_ - qi::eol) 
+    >> qi::eol
+    ;
+
+  inQuotes =
+    '"'
+    > *(qi::char_ - '"')
+    > '"'
+    ;
+
+  token =
+    +qi::ascii::graph
+    ;
+
+  csvToken =
+    +(qi::ascii::graph - qi::char_(','))
+    ;
```

Comment:
Dead code?  I don't think this is used anywhere, unless you end up using it where I mentioned in one of my other comments.